.editorconfig

```diff
@@ -0,0 +1,17 @@
+# editorconfig.org
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
```

Comment:
So I changed everything to tabs.. if you don't mind we can keep the 2 and merge it.. I already squashed it.